coursedata/adventures/de.yaml

```diff
@@ -603,164 +609,195 @@ adventures:
+                example_code_2: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    angles = ask 'How many angles should I draw?'
-                    angle = 360 / angles
-                    repeat angle times
+                    Ecken = ask 'Wie viele Ecken soll die Form haben?'
+                    Winkel = 360 / Ecken
+                    repeat Ecken times
                         turn _
                         forward _
                     ```
                 start_code: |
-                    hoeken = ask 'How many angles should I draw?'
+                    Ecken = ask 'Wie viele Ecken soll die Form haben?'
             7:
                 story_text: |
-                    ## Let's draw
-                    In this level you can repeat one line of code with `repeat`.
+                    ## Lass uns zeichnen!
+                    In diesem Level kannst Du mit dem Befehl `repeat` eine Linie mehrfach zeichnen.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    print 'Draw figures'
+                    print 'Zeichne Formen'
                     repeat 3 times forward 10
                     ```
                 start_code: |
-                    print 'Draw figures'
+                    print 'zeichne Formen'
                     repeat 3 times forward 10
 
             6:
                 story_text: |
-                    ## Let's draw
-                    In this level you can use calculations to draw different figures.
-                    You may have learned in school that turning a full circle is 360 degrees. If not, now you know!
-                    That's why you also use 90 degrees for a square. 360 divided by 4 is 90.
-                    Now that we can do math with Hedy, we can draw all the figures we want!
+                    ## Lass uns zeichnen!
+                    In diesem Level nutzen wir Berechnungen um verschiedene Formen zu zeichnen.
+                    Du hast vielleicht schon gelernt, dass eine komplette Umdrehung 360° sind.
+                    Falls nicht, dann weißt Du es jetzt!
+
+                    Deswegen benutzen wir immer 90° für ein Rechteck.
+                    360 geteilt durch 4 ist 90.
+
+                    Jetzt, wo wir Mathe mit Hedy machen können, können wir alle Formen zeichnen, die wir uns wünschen!
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    angles = ask 'How many angles do you want?'
-                    angle = 360 / angles
+                    Ecken = ask 'Wie viele Ecken soll die Form haben?'
+                    Winkel = 360 / Ecken
```

Comment:
again, should we put "Anzahl" ("number of …") into the variable name for the number of corners?

Reply:
See above. Wouldn't complicate things.